Setting-envars-docs #3582 (#7400)
opensearch-project/documentation-website: _install-and-configure/configuring-opensearch/index.md

## Specifying settings as environment variables

> You can specify environment variables in different ways.

Reviewer: ```suggestion
You can specify environment variables in the following ways.
```

> ### Arguments as start up

Reviewer 2: Should "as" be "at"? Also, if "startup" is a noun, it should be one word with no space.

You can specify environment variables as arguments using `-E` when launching OpenSearch:

```bash
./opensearch -Ecluster.name=opensearch-cluster -Enode.name=opensearch-node1 -Ehttp.host=0.0.0.0 -Ediscovery.type=single-node
```
{% include copy.html %}

### Directly in the shell environment

> You can configure the environment variables directly in shell before starting OpenSearch:

Reviewer: ```suggestion
You can configure the environment variables directly in a shell environment before starting OpenSearch, as shown in the following example:
```

```bash
export OPENSEARCH_JAVA_OPTS="-Xms2g -Xmx2g"
export OPENSEARCH_PATH_CONF="/etc/opensearch"
./opensearch
```
{% include copy.html %}

### Systemd service file

> When running OpenSearch as a service managed by systemd, you can specify environment variables in the service file:

Reviewer: ```suggestion
When running OpenSearch as a service managed by `systemd`, you can specify environment variables in the service file, as shown in the following example:
```

```bash
# /etc/systemd/system/opensearch.service.d/override.conf
[Service]
Environment="OPENSEARCH_JAVA_OPTS=-Xms2g -Xmx2g"
Environment="OPENSEARCH_PATH_CONF=/etc/opensearch"
```

> After creating or modifying the file, reload the systemd configuration and restart the service:

Reviewer: ```suggestion
After creating or modifying the file, reload the systemd configuration and restart the service using the following command:
```

```bash
sudo systemctl daemon-reload
sudo systemctl restart opensearch
```
{% include copy.html %}

### Docker environment variables

> When running OpenSearch in Docker, you can specify environment variables using the -e option with Docker run:

Reviewer: ```suggestion
When running OpenSearch in Docker, you can specify environment variables using the -e option with Docker run, as shown in the following command:
```

> When running OpenSearch in Docker, you can specify environment variables using the -e option with Docker run, as shown in the following command:

Reviewer 2: Should it be "the `-e` option with `docker run`"?
